elasticdl/pkg/ps/model.go

```diff
@@ -49,7 +49,7 @@ func (model *Model) SetEmbeddingTableInfo(info *proto.EmbeddingTableInfo) {
 
 // InitFromModelPB inits the model from model PB
 func (model *Model) InitFromModelPB(pb *proto.Model) error {
-	for _, v := range pb.EmbeddingTableInfo {
+	for _, v := range pb.EmbeddingTableInfos {
```

Comment:
`Info` has no plural form.

Reply:
How about change `Info` to `Desc`? `Desc` is short for `Description`

Reply:
descriptions/attributes/properties/information. Hard to choose a better one.

We can keep to use `EmbeddingTableInfos`, since it can be understood.